<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="transport_abstract">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>71</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="DoIP">
     <attribute name="title">
      <string>DoIP_Server</string>
     </attribute>
     <widget class="QComboBox" name="Net_Select">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Server Addr</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="doip_server_addr">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>0x0040</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Clent Addr</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="doip_client_addr">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>10</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>0x0E80</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="DoCAN">
     <attribute name="title">
      <string>DoCAN Server</string>
     </attribute>
     <widget class="QCheckBox" name="is_FD">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>FD</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="CAN_Server_Addr">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>0x0040</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Server Addr</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Clent Addr</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="CAN_CLient_Addr">
      <property name="geometry">
       <rect>
        <x>338</x>
        <y>10</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>0x0E80</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="tracebox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>471</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="uds_init">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="config_file">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Config File</string>
     </property>
    </widget>
    <widget class="QLabel" name="config_file_label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>271</width>
       <height>25</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Sunken</enum>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="init_server">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Init Server</string>
     </property>
    </widget>
   </widget>
   <zorder>uds_init</zorder>
   <zorder>transport_abstract</zorder>
   <zorder>tracebox</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>492</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
